agasc/supplement/magnitudes/update_mag_supplement.py

```diff
@@ -324,28 +328,23 @@ def do(output_dir,
         agasc_ids = np.intersect1d(agasc_ids, star_obs_catalogs.STARS_OBS['agasc_id'])
 
     # set start/stop times and agasc_ids
-    if whole_history:
+    if whole_history or agasc_ids is not None:
         if start:
-            logger.warning('Ignoring --start argument from commant line (--whole-history)')
+            logger.warning('Ignoring --start argument from commant line')
```

Comment:
In general, incompatible command line arguments should raise an exception,  not warn. Otherwise the program is guessing what the user really wanted.

Reply:
Fixed